datemath/__init__.py

```diff
@@ -1,9 +1,18 @@
-from .helpers import parse, DateMathException
+from __future__ import annotations
```

Comment:
This allows the use of ` | ` and symbols that won't be present at runtime in annotations.